src/ui/components/molecules/side-panel/panel/SPanel.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import type { IProps } from './types/IProps';
+import LogoFooter from '../../../atoms/logo-footer/LogoFooter';
+import Tags from '../../../atoms/Tags/Tags';
+import './SPanel.scss';
+import logo from '../../../../../assets/icons/logo.svg?raw';
+
+function SPanel({ color = 'primary' }: IProps) {
+  const tags = ['workings', 'optimism', 'meaning', 'promoting'];
+  const getColorClassName = () => {
+    return color === 'primary' ? 'panel--primary' : 'panel--secondary';
+  };
+
+  return (
+    <div className={`panel ${getColorClassName()}`} data-testid="s-panel">
+      <LogoFooter icon={logo} color="primary" />
+      <Tags labels={tags} />
```

Comment:
Hello, it is my fault for not checking the Tags well, this component should be current as simple links, we will use navigation by # to go through the articles.
I'm thinking of another approach, It is probably it was necessary to implement a pub-sub system to communicate with all entry applications, let me think a little bit, but for a while please modify this component